coderd/database/queries/workspaceapps.sql

```diff
@@ -28,10 +28,11 @@ INSERT INTO
         healthcheck_interval,
         healthcheck_threshold,
         health,
-        display_order
+        display_order,
+		hidden
```

Comment:
nit: extra indent

Reply:
oops, looks like it's been inserted as tabs instead of spaces which is why I didn't notice this (my tabs are set to 4 spaces instead of 8)